bin/node/runtime/src/lib.rs

```diff
@@ -208,6 +211,45 @@ impl pallet_bridge_eth_poa::Trait for Runtime {
 	type OnHeadersSubmitted = ();
 }
 
+impl pallet_bridge_currency_exchange::Trait for Runtime {
+	type PeerBlockchain = exchange::EthBlockchain;
+	type PeerMaybeLockFundsTransaction = exchange::EthTransaction;
+	type RecipientsMap = sp_bridge_eth_poa::exchange::AsIsRecipients<AccountId>;
+	type Amount = Balance;
+	type CurrencyConverter = sp_bridge_eth_poa::exchange::AsIsCurrencyConverter<Balance>;
+	type Airdrop = Airdrop;
+}
+
+pub struct Airdrop;
+
+impl sp_bridge_eth_poa::exchange::Airdrop for Airdrop {
+	type Recipient = AccountId;
+	type Amount = Balance;
+
+	fn drop(recipient: Self::Recipient, amount: Self::Amount) -> sp_bridge_eth_poa::exchange::Result<()> {
+		<pallet_balances::Module<Runtime> as Currency<AccountId>>::deposit_into_existing(&recipient, amount)
```

Comment:
```suggestion
		<pallet_balances::Module<Runtime> as Currency<AccountId>>::deposit_creating(&recipient, amount)
```
IMHO we should not fail if the account does not exist. Sending bridged funds might be the first encounter with substrate blockchain, we should not require any prior interaction.

Reply:
On the second thought I think we should rather use `issue` + `resolve_creating` to make sure that the change in total issuance is reflected.

Reply:
On the third thought, if we plan to introduce a two-way bridge into existing networks that have a value already (like Kusama) we should not change (mint/bunr) the total issuance but rather have some fixed pool that we can use to transfer the funds from/to. On the other chain locked tokens could be redeemed by the ones that top-up that pool.

Overall it seems that this will rather need to be configurable, so my other comment about using `Currency` directly (instead of `Airdrop`) should be disregarded.